### PR TITLE
feat: add oraclelinux7-nodejs:14-oracledb

### LINKED ISSUE #1940: Create oraclelinux7-nodejs:14-oracledb
https://github.com/oracle/docker-images/tree/main/OracleLinuxDevelopers is missing a "oraclelinux7-nodejs:14-oracledb" file. There is a node-oracledb RPM  for Node.js 14 at  https://yum.oracle.com/repo/OracleLinux/OL7/developer/nodejs14/x86_64/

### PR DESCRIPTION
**Pull request by Djelibeybi**
Resolves #1940.

Signed-off-by: Avi Miller <avi.miller@oracle.com>

### CODE REVIEW

File: OracleLinuxDevelopers/oraclelinux7/nodejs/14-oracledb/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+FROM oraclelinux:7-slim
+
+RUN yum -y install oracle-nodejs-release-el7 oracle-instantclient-release-el7 && \
+    yum -y install nodejs node-oracledb-node14 && \
+    rm -rf /var/cache/yum/*
+
```
Comment (cjbj) [inline OracleLinuxDevelopers/oraclelinux7/nodejs/14-oracledb/Dockerfile:8]: Add `ENV NODE_PATH=/usr/lib/node_modules` like the other Node.js oracledb Dockerfiles. (Untested; I'll test later).

Comment (Djelibeybi) [reply OracleLinuxDevelopers/oraclelinux7/nodejs/14-oracledb/Dockerfile:8]: Good catch.

**Review by totalamateurhour — APPROVED** [review]
LGTM

**Review by AmedeeBulle — APPROVED** [review]
LGTM

**Pull request merged**